jina/clients/python/__init__.py

```diff
@@ -101,11 +101,17 @@ def call_unary(self, data: Union['jina_pb2.Document', bytes], mode: ClientMode)
         req_iter = getattr(request, str(self.mode).lower())(**kwargs)
         return self._stub.CallUnary(next(req_iter))
 
-    def call(self, callback: Callable[['jina_pb2.Message'], None] = None, **kwargs) -> None:
+    def call(self, *args, **kwargs) -> None:
```

Comment:
Here, mypy complains about signature mismatch of the base and overloading method.
Since the signature of the base method is more extensible and less likely to be changed in future, I decided to tweak the signature of the overloading method.

Reply:
This is a sensitive part that I would not change in this scope

Reply:
ok